Bootstrap practice/indexpractice.html

```diff
@@ -51,7 +51,7 @@
 <div class="row main-row fill">
   <div class="col-sm-3">
     <div class="sidebar-nav">
-      <div class="navbar navbar-default" role="navigation">
+      <div class="col-sm-3 navbar navbar-default" role="navigation">
```

Comment:
@bambiwants this makes the sticky block's width the same as the grid system, even though a `position:fixed` element totally fucks off from the grid system. It's sorta a hack. Sorta.